scope.go

```diff
@@ -66,7 +66,7 @@ type scope struct {
 	defaultBuckets Buckets
 
 	registry *scopeRegistry
-	quit     chan struct{}
+	status   *scopeStatus
```

Comment:
Since it does not need to be passed to child scopes I can do this, I'll update it.